modules/ListView.js

```diff
@@ -171,7 +170,8 @@
      *   @param [args.modelId='cid'] {String} - model property used as identifier for a given model. This property is saved and used to find the corresponding view.
      *   @param [args.childModel='model'] {String} - name of the model argument passed to the child view during initialization
      */
-    initialize: function(args) {
+    constructor: function(args) {
```

Comment:
note that this requires an upgrade.  If a view set its collection before calling the ListView initialize it will now either have to convert to constructors or use setCollection(); - just make sure this gets into the release notes.